posts/hybrid-cloud-cloud-responsibly.md

```diff
@@ -0,0 +1,94 @@
+
+The arrowed lines represent the request flow through the deployed application. The user interface is a single page application (SPA) that obtains content from the gateway and submits REST calls for data to the gateway. The gateway forwards requests to Fabio which are then load balanced and routed to the appropriate microservice to be serviced. Additionally, microservices can leverage Kafka streams to support queuing and asynchronous processing for applications having those needs.
+
+<h2>Example</h2>
+Hybrid Cloud is about running your infrastructure where it makes sense. For example, the diagram below shows an application in AWS so that it’s fault tolerant, and it uses Fabio which routes to microservices either on-prem or in the cloud depending on whether it’s a compute-heavy request (cloud) or something simple like a lookup (on-prem). While the federated directory and user authentication is kept private on-prem.
+
+
+![alternate text](https://github.com/ippontech/blog-usa/blob/master/images/2019/08/IpponWay-Hybrid-Cloud-Example.png)
+
+
+<h2>Software Selection</h2>
+Ippon selected the following software for our solution:
+
+- Scheduler/Orchestration - Nomad
+- Service Discovery - Consul
+- Configuration/Secrets - Vault, Consul
+- Health/Performance Monitoring - Prometheus
+- App Router/Load Balancer - Fabio
+- Role-based Access Control (Authorization) - Spring OAuth - Keycloak
+- Authentication - Keycloak OpenId Connect
+- Code generation - JHipster
+- Data modeling - JHipster JDL
+- Microservice Framework - Spring Boot
+- Front-end development framework/language - Angular/JavaScript/TypeScript
+- Back-end/Microservice framework/language - Spring Boot/Java
+- User Account Management - Keycloak
+- Cloud Deployment - Nomad, Ansible, Docker
+- Queuing/Streaming - Kafka
+
+A key aspect of the above software selection was the preference for open source software over commercial software. For example, this guided the choice of Keycloak over Okta for
+authentication and authorization services. HashiCorp software (Nomad, Consul) was selected to obtain the abstraction layer to insulate the solution from the specifics of individual cloud providers and on-prem. JHipster was selected because of our company’s expertise in using it to quickly produce enterprise grade applications. The remaining software selections were guided by popularity in the industry and proven reliability.
+
+While our selections represent Ippon’s preferences for building this solution, our understanding of the roles of each component allow us to interchange them as necessary to fit into specific environments dictated by the needs and desires of our clients.
```

Comment:
I would like to highlight this sentence somehow in the blog. Whether that means a different font or maybe a different background....I don't know.